contracts/interfaces/IVaultFactory.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Interface for a vault factory contract
+ */
+interface IVaultFactory {
```

Comment:
note: we _could_ implement the entire AssetWrapper interface here, but its so much worse than just sending assets into the vault it felt like unnecessary cruft